package.json

```diff
@@ -40,7 +41,9 @@
     "html-webpack-plugin": "^2.30.1",
     "jest": "^20.0.4",
     "lerna": "^2.0.0",
+    "node-sass": "^4.5.3",
```

Comment:
the reason why I install `node-sass` and `sass-loader` is that maybe it will be used for package 
`source` some day.